umpost/conversion_driver_esm1p5.py

```diff
@@ -292,11 +299,15 @@ def convert_esm1p5_output_dir(esm1p5_output_dir):
                             "Otherwise report full stack trace."
                         )
                         )
+    parser.add_argument("--delete_ff", "-d", action="store_true",
```

Comment:
This is a good setting as the default operation is safe (non-destructive). This places the onus on users to consider deletion.

The arg should be `--delete-ff` (hyphen) as `argparse` automatically converts the hyphen to an underscore in the args data structure --> `arg.delete_ff`.

Reply:
Ah thanks, I hadn't realised that! I've just fixed this up.

Reply:
That detail is _buried_ somewhere in the `argparse` docs.